java/sage/media/image/RawImage.java

```diff
@@ -25,7 +25,6 @@
   /** Creates a new instance of RawImage */
   public RawImage(int width, int height, byte[] data, boolean hasAlpha, int stride)
   {
-    if (!ImageLoader.EMBEDDED) throw new UnsupportedOperationException("RawImage objects with byte[] backed data is only supported on embedded.");
```

Comment:
Done. Also removed the dataArr field and getDataArr() methods since they were unused after removing the constructor.